acceptance/testdata/release/upload-download.txtar

```diff
@@ -0,0 +1,25 @@
+# Create a repository with a file so it has a default branch
+exec gh repo create $ORG/$SCRIPT_NAME-$RANDOM_STRING --add-readme --private
+
+# Defer repo cleanup
+defer gh repo delete --yes $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Clone the repo
+exec gh repo clone $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Create a release in the repo
+cd $SCRIPT_NAME-$RANDOM_STRING
+exec gh release create v1.2.3 --notes 'awesome release' --latest
+
+# Upload an asset to the release
+exec gh release upload v1.2.3 ../asset.txt
+stdout ''
```

Comment:
🤔 have you considered checking `stderr` instead of `stdout`, here? I took a quick look at `uploadRun()` and it seems the only time that `stdout` won't be empty is for the [TTY use case](https://github.com/cli/cli/blob/8b0b0ce1843fcdcf662e655ddc05a1e10fd4a747/pkg/cmd/release/upload/upload.go#L118-L123). Thus, I think if this errors, this test may still pass.

Reply:
Or does `exec gh release upload v1.2.3 ../asset.txt` catch errors for us already? If so, do we need this check?

Reply:
Ah, `exec` definitely [is an assertion](https://bitfieldconsulting.com/posts/test-scripts#:~:text=This%20exec%20statement%2C%20though%2C%20is%20more%20than%20just%20a%20way%20of%20running%20programs%3B%20it%E2%80%99s%20also%20an%20assertion.%20The%20program%20we%20execute%20must%20succeed%3A%20that%20is%2C%20its%20exit%20status%20must%20be%20zero.%20Otherwise%2C%20the%20exec%20assertion%20fails.)!

> This exec statement, though, is more than just a way of running programs; it’s also an assertion. The program we execute must succeed: that is, its exit status must be zero. Otherwise, the exec assertion fails.

Reply:
So what is the `stdout ''` checking for?

Reply:
@BagToad and I discussed these and agreed to remove them because they were asserting a bit more than was necessary.

Reply:
Ah, cool. I thought I might be missing something 👍